xarray/core/duck_array_ops.py

```diff
@@ -48,6 +47,15 @@ def get_array_namespace(x):
         return np
 
 
+def einsum(*args, **kwargs):
+    if module_available("opt_einsum"):
```

Comment:
Can we add a global option like `use_bottleneck` to disable this (`True` by default)? I get that using `opt_einsum` has advantages, but it usually is very annoying if a library uses different code paths based on installed packages without having a way to opt-out in code (among other things, this could be used for testing)
```suggestion
    if use_opt_einsum and module_available("opt_einsum"):
```

Reply:
I was negative on this suggestion, but now that I see `pint` doesn't work, perhaps we do need it :/